src/components/ui/AudioWaveform.tsx

```
@@ -0,0 +1,197 @@
+  return (
+    <div className={cn("space-y-2", className)}>
+      <div className="flex items-center justify-between text-xs text-muted-foreground font-mono px-1">
+        <span>Waveform (Original Cut)</span>
+        <span>Click to seek playhead</span>
+      </div>
+
+      <div
+        ref={containerRef}
+        onClick={handleContainerClick}
+        className="relative w-full h-24 bg-background/50 border border-border rounded-xl flex items-center justify-between px-3 cursor-pointer select-none overflow-hidden hover:border-border/80 transition-colors duration-200"
+      >
```

Biome (2.4.15)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)